Client/src/wrapWdkService.js

```diff
@@ -26,5 +26,14 @@ export default wdkService => ({
     useCache: false,
     method: 'get',
     path: '/site-messages'
-  })
+  }),
+  getUser: () => {
+    let user = wdkService.getCurrentUser();
+    if (useEda) {
+      user.properties.skipStudyPermissions = false;
+    } else {
+      user.properties.skipStudyPermissions = true;
+    }
+    return user;
+  }
```

Comment:
`wdkService.getCurrentUser()` returns a `Promise`, which requires slightly different handling. Also, this method should also be named `getCurrentUser` (I misspoke in our original conversation`.

```suggestion
  getCurrentUser: async () => {
    let user = await wdkService.getCurrentUser();
    if (useEda) {
      user.properties.skipStudyPermissions = false;
    } else {
      user.properties.skipStudyPermissions = true;
    }
    return user;
  }
```

Reply:
Awesome! Thanks @dmfalke !